resources/lightspeed-plugins.yaml

```diff
@@ -0,0 +1,19 @@
+plugins:
```

Comment:
Our other plugins are kept in the `dynamic-plugins` dir, should we store this there too for consistency?

Reply:
Ah yes that is a good point, at the end of the day it doesn't matter if we leave it in the `/resources` dir but maybe for clarity to separate plugins from other config files we could put it into a `/optional-plugins` dir?

wdyt @jrichter1 @michael-valdron , I am indifferent though and it doesn't _really_ matter

Reply:
> maybe for clarity to separate plugins from other config files we could put it into a `/optional-plugins` dir?

I'm liking this idea so we can still seperate the plugin lists from the kubernetes/openshift resource specs.